compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -90,12 +89,31 @@ import qualified Text.PrettyPrint.ANSI.Leijen      as PP
 -- Commands
 --------------------------------------------------------------------------------
 
-cmdIde :: Mod CommandFields Command
+data CommandName = Build
```

Comment:
If you move this to the next line, i.e., something like
```
data CommandName
  = Build
  | …
```
you won’t create large diffs if the length of `CommandName` changes.